.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @diogomatsubara @joelrebel @mmlb @turegano-equinix 
```

Comment:
Could you drop Diogo from this list and include @splaspood

Reply:
done